src/components/ContactForm/ContactForm.tsx

```diff
@@ -3,7 +3,11 @@ import { useState } from 'react';
 import styles from './ContactForm.module.css';
 import { createShortId } from '../../serviсes/shortId';
 
-export default function ContactForm({ onSubmit }) {
+interface ContactFormProps {
+  onSubmit: Function
+}
+
+export default function ContactForm({ onSubmit }: ContactFormProps) {
   const [contact, setContact] = useState({
```

Comment:
please provide type for this `useState`

Reply:
Type for `useState` has been provided as interface ContactState. I tried to use the ContactData type, but the editor shows an error that the `const contact` in the useState doesn't have an `id`.

Reply:
remove `ContactState` and use `ContactData`